VideoLocker/build.gradle

```diff
@@ -180,6 +182,7 @@ android {
         // test configuration
         testApplicationId 'org.edx.mobile.test'
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "com.facebook.testing.screenshot.ScreenshotTestRunner"
```

Comment:
Would definitely like to look into this more but that was another rabbit hole I wasn't ready to get into.